packages/rx_bloc_list/pubspec.yaml

```diff
@@ -12,19 +12,18 @@ dependencies:
   collection: ^1.17.1
   flutter:
     sdk: flutter
-  flutter_rx_bloc: ^6.0.0
-  rx_bloc: ^5.0.0
+  flutter_rx_bloc: ^6.1.1
```

Comment:
Do we really want to need the 6.1.1 or 6.1.0 works fine? The patch number requirement needs to be specified only when absolutely needed